### PR TITLE
:tada: Add "Penpot free" label

### PR DESCRIPTION
**Pull request by pabloalba**
Implements https://tree.taiga.io/project/penpot/task/9508

### CODE REVIEW

File: frontend/src/app/main/ui/dashboard/sidebar.cljs [hunk 1]
```diff
@@ -939,9 +954,21 @@
...
+         #(on-click :settings-profile %))
+
+        on-power-up-click
+        (mf/use-fn
+         (fn []
+           (dom/open-new-window "https://penpot.app/pricing")))]
 
     [:*
+     [:div {:class (stl/css :upgrade-plan-section)
+            :on-click on-power-up-click}
```
Comment (belen-albeza) [inline frontend/src/app/main/ui/dashboard/sidebar.cljs]: This looks like it should be a `<button>` tag.

File: frontend/src/app/main/ui/dashboard/sidebar.scss [hunk 2]
```diff
@@ -391,3 +401,29 @@
...
+
+.upgrade-plan-section {
+  display: flex;
+  justify-content: space-between;
+  border: $s-1 solid var(--color-background-quaternary);
+  border-radius: $s-8;
+  padding: $s-12;
+  margin: $s-12 $s-8 $s-12 $s-12;
+  color: var(--color-foreground-secondary);
+  cursor: pointer;
```
Comment (belen-albeza) [inline frontend/src/app/main/ui/dashboard/sidebar.scss]: The `$s-xx` tokens are deprecated, please use the semantic ones defined at `ds/spacing.scss` (i.e. `--sp-m`, `--sp-l`, etc.

File: frontend/src/app/main/ui/dashboard/sidebar.scss [hunk 3]
```diff
@@ -391,3 +401,29 @@
...
+}
+
+.no-limits {
+  font-size: $fs-12;
+}
+
+.power-up {
+  font-size: $fs-12;
+  color: var(--color-accent-tertiary);
+}
```
Comment (belen-albeza) [inline frontend/src/app/main/ui/dashboard/sidebar.scss]: We have a `useTypography` mixin available at `typography.scss`. Please use it with one of the typographies of the DS.

File: frontend/src/app/main/ui/dashboard/sidebar.cljs (hunk 1 above)
Comment (pabloalba) [reply frontend/src/app/main/ui/dashboard/sidebar.cljs]: done

File: frontend/src/app/main/ui/dashboard/sidebar.scss (hunk 2 above)
Comment (pabloalba) [reply frontend/src/app/main/ui/dashboard/sidebar.scss]: done

File: frontend/src/app/main/ui/dashboard/sidebar.scss (hunk 3 above)
Comment (pabloalba) [reply frontend/src/app/main/ui/dashboard/sidebar.scss]: done

**Review by belen-albeza — APPROVED** [review]

**Pull request merged**